abc.templates/base-workflows/testdata/golden/basic/test.yaml

```diff
@@ -0,0 +1,16 @@
+api_version: cli.abcxyz.dev/v1beta3
+kind: GoldenTest
+inputs:
+    - name: guardian_state_bucket
+      value: my-guardian-state-bucket
+    - name: terraform_directory
+      value: .
+    - name: terraform_version
+      value: 1.7.4
+    - name: guardian_wif_provider
+      value: my-guardian-wif-provider
+    - name: guardian_service_account
+      value: my-guardian-service-account@abcxyz-my-project.iam.gserviceaccount.com
+builtin_vars:
+    - name: _git_tag
+      value: my-git-tag
```

Comment:
Can we make this something more accurate like `v1.0.0`, the issue we ran into was with the v prefix in our templates.